cli/lib/kontena/cli/whoami_command.rb

```diff
@@ -10,10 +10,16 @@ def execute
     end
 
     require_api_url
-    puts "Master: #{settings['server']['url']}"
     token = require_token
-    response = client(token).get('user')
-    puts "User: #{response['email']}"
+
+    puts "Master: #{settings['server']['url']}"
+
+    grid = client(require_token).get("grids/#{current_grid}")
```

Comment:
I think we can skip fetching grid details. `current_grid` already is the grid name?